kedro/framework/cli/starters.py

```diff
@@ -288,11 +392,12 @@ def _get_cookiecutter_dir(
                 f" Specified tag {checkout}. The following tags are available: "
                 + ", ".join(_get_available_tags(template_path))
             )
-        official_starters = sorted(_STARTER_ALIASES)
-
+        official_starters = sorted(_OFFICIAL_STARTER_SPECS)
         raise KedroCliError(
             f"{error_message}. The aliases for the official Kedro starters are: \n"
-            f"{yaml.safe_dump(official_starters)}"
+            f"""{yaml.safe_dump(
+                [starter.format() for starter in official_starters],
+                sort_keys=False)}"""
```

Comment:
This should just be aliases, so `official_starters.keys()`.

Reply:
In fact it is now a dictionary, so after sorting it is just a list of key!